infra/terraform/output.tf

```diff
@@ -0,0 +1,4 @@
+output "owtf_url" {
```

Comment:
Assigning and using the raw IP for the Ec2 machine is very insecure - we should have a load balancer in front that proxies request to the EC2 machine which is in a private VPC, so it is never exposed to the internet